coffee-tool/src/main/java/hu/icellmobilsoft/coffee/tool/utils/string/StringUtil.java

```diff
@@ -216,4 +216,21 @@ public static String replaceAllIgnoreCase(String text, String regex, String repl
         return text;
     }
 
+    /**
+     * Mask authentication credentials in URI connection string.
+     *
+     * This function replaces the "username:password" part between the "//" and "@" signs in the URI with "*:*". E.g.:<br>
+     * "mongodb://username:password@localhost:12345" becomes "mongodb://*:*@localhost:12345"
+     *
+     * @param uri
+     *            the uri to mask
+     * @return masked URI or unchanged if invalid input
+     */
+    public static String maskUriAuthenticationCredentials(String uri) {
+        if (StringUtils.isBlank(uri)) {
+            return uri;
+        }
+        String regex = "//.*@";
+        return uri.replaceAll(regex, "//*:*@");
```

Comment:
regex-re hasznalunk processor cache-t, azt itt is jó lenne alkalmazni

Reply:
rendben